feature/gribi/otg_tests/gribi_static_route_on_lemmings_test/gribi_static_route_on_lemmings_test.go

```diff
@@ -0,0 +1,188 @@
+package basic_static_route_support_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"google3/third_party/open_traffic_generator/gosnappi/gosnappi"
```

Comment:
Import packages from github.com, google3 is for internal usage. Same applies for all the imports used in this file. Refer internal/otg_helpers/otg_config_helpers/otgconfighelpers.go for reference

Reply:
import file paths are updated to gitub